assembl/static2/js/app/components/cookies/cookiesSelector.jsx

```diff
@@ -50,7 +50,12 @@ const CookiesSelector = ({
                     handleCategorySelection(category);
                   }}
                 >
-                  <span className={classnames('assembl-icon-right-dir', { 'active-arrow': isActiveKey })} />
+                  <span
+                    className={classNames({
+                      'assembl-icon-down-dir': isActiveKey || show,
+                      'assembl-icon-right-dir': !isActiveKey || !show
+                    })}
+                  />
```

Comment:
Can't we use the same assembl-icon-angle-down and assembl-icon-angle-right here to have the same arrows like in the navigation?

Reply:
sure